trademapper/js/config.js

```diff
@@ -0,0 +1,44 @@
+define({
+	// global colour settings
+	colours: {
+		COUNTRY: "#DFDFDF",
+		COUNTRY_TRADING: "#B1C73E",
+		COUNTRY_BORDER: "#FFF",
+		OCEAN: "#FFF",
+		DISPUTED: "#AAA",
+		ARROW_WIDE: "rgba(255, 255, 255, 0.5)",
+		ARROW_NARROW: "rgba(0, 0, 0, 1.0)",
+		ARROW_NARROW_PLAIN: "rgba(102, 102, 102, 1.0)",
+
+		// colours for nodes; if these are changed here, the corresponding
+		// variables at the top of the trademapper.scss file should also be edited
```

Comment:
This is ok for now, but in the medium term we should remove them from the sass altogether.

If the colours need to be edited in two places anyway, this means using CSS to define the colours is a bit pointless. So we might as well remove them from the sass and set them directly in Javascript. Also looking ahead here: at some point we'll want to allow users to customise colours, at which point we will set them in Javascript anyway.

We'll need to think this through so as I said, this is ok for now :)

Reply:
Agreed. I did consider taking the values from config and manually setting CSS styles, but thought the usefulness of that now (when users can't editor colours anyway) outweighed the effort required.